ethereum-verifier/Readme.md

```diff
@@ -0,0 +1,9 @@
+# Ethereum Stark Verifier
+## Install
+`npm init`
```

Comment:
The project should already be initialized and have a `package.json`. Just running `npm install` should be enough.